### PR TITLE
Fixes #15829: Cleanup roles in packaging

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/15829

### CODE REVIEW

File: rudder-webapp/SOURCES/rudder-node-to-relay [hunk 1]
```diff
@@ -78,13 +78,7 @@ fi
 
 # Override any server values with those from ${RUDDER_ROLES_FILE}
```
Comment (Fdall) [inline rudder-webapp/SOURCES/rudder-node-to-relay]: Maybe we can removed this now ?

Comment (Fdall) [reply rudder-webapp/SOURCES/rudder-node-to-relay]: It is not used anywhere

**Comment by peckpeck** [comment]
PR updated with a new commit

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Pull request merged**